### PR TITLE
Added environmental variables

### PR DESCRIPTION
**Pull request by sinisbad123**

### CODE REVIEW

**Comment by sinisbad123** [comment]
Please see implementation as recommended here:

https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env

Let me know if you guys wanted to change the name of the env variables

**Review by KostyaTookitaki — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH_URL=http://52.191.8.13:7080/api/v1
+REACT_APP_API_URL=http://52.191.8.13:7080/api/v1
```
Comment (KostyaTookitaki) [inline .env:2]: for security reasons file `.env`/`.env. development`/`.env.production`/etc... should be part of `.gitignore` to not expose secret keys or something like that. Currently we don't have secrets as far as I understand but they can be needed soon.
I prefer to have file called `.env.example` with non-secret values listed along with placeholders for secrets like so:

```
REACT_APP_AUTH_URL=http://52.191.8.13:7080/api/v1
REACT_APP_API_URL=http://52.191.8.13:7080/api/v1
REACT_APP_ANOTHER_NON_SECRET_KEY=real-key
REACT_APP_SOME_SECRET_KEY=<here-should-be-secret-key>
```

so developers can copy this file to `.env`/`.env. development`/`.env.production`/etc... and replace `<here-should-be-secret-key>` with the real key.

So basically `.gitignore` should contain something like

```
.env.*
!.env.example
```

**Review by KostyaTookitaki — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (sinisbad123) [reply .env:2]: Hey man,

This is just a template only, for now this'll do.

I don't want to implement other .env files yet, since i believe this repo is public, let's discuss offline

Comment (KostyaTookitaki) [reply .env]: hey 👋 I requested this change exactly because this repo is a template for others to follow, boilerplate project is needed to quickly start development without rethinking common stuff. You don't need to implement other .env files - they are supported by default by create-react-app, we just need to set up right `.gitignore` file and to have example .env file which can be copied to whatever .env.* file is needed

Comment (sinisbad123) [reply .env]: Oh, okay!

**Review by KostyaTookitaki — APPROVED** [review]
